tavla/app/(admin)/tavler/[id]/rediger/components/ActionsMenu/index.tsx

```diff
@@ -13,38 +16,56 @@ function ActionsMenu({
 }) {
     return (
         <>
-            <OverflowActionsMenu board={board} folderid={folderid} />
-            <ButtonsMenu board={board} folderid={folderid} />
+            <ActionsMenuDesktop board={board} folderid={folderid} />
+            <ActionsMenuPhone board={board} folderid={folderid} />
         </>
     )
 }
 
-function OverflowActionsMenu({
+function ActionsMenuDesktop({
     board,
     folderid,
 }: {
     board: BoardDB
     folderid?: FolderId
 }) {
     return (
-        <div className="hidden md:flex">
+        <div className="hidden flex-row gap-4 sm:flex md:items-center">
+            <Open bid={board.id} type="button" />
+            <RefreshButton board={board} />
+            <Delete board={board} type="button" />
             <OverflowMenu placement="bottom-left">
-                <DuplicateBoard board={board} folderid={folderid} />
+                <DuplicateBoard
+                    board={board}
+                    folderid={folderid}
+                    type="menuitem"
+                />
             </OverflowMenu>
         </div>
     )
 }
 
-function ButtonsMenu({
+function ActionsMenuPhone({
     board,
     folderid,
 }: {
     board: BoardDB
     folderid?: FolderId
 }) {
     return (
-        <div className="flex flex-col gap-4 md:hidden md:flex-row md:items-center">
-            <DuplicateBoard board={board} folderid={folderid} />
+        <div className="hidden flex-row max-sm:flex md:hidden">
```

Comment:
Kunne denne også vært sånn her? 
```suggestion
        <div className="flex-row sm:flex md:hidden">
```